runtime/src/bank.rs

```diff
@@ -3457,7 +3514,7 @@ mod tests {
         bank = Arc::new(Bank::new_from_parent(&bank, &Pubkey::default(), 49));
         assert_eq!(
             bank.rent_collection_partitions(),
-            vec![(14, 31, 32), (0, 17, 64)]
+            vec![(14, 31, 32), (0, 0, 64), (0, 17, 64)]
```

Comment:
It's actually not great that these seem to be hard-coded to what the epoch schedule is set to. If I change `MINIMUM_SLOTS_PER_EPOCH` to something else, then all these tests will fail.

Reply:
I know it... I guess there is other tests relying on `MINIMUM_SLOTS_PER_EPOCH`. Anyway, we can always improve tests regarding this later...